packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -2078,7 +2098,6 @@ const VideoPlayerBase = class extends React.Component {
 					// This captures spotlight focus for use with 5-way.
 					// It's non-visible but lives at the top of the VideoPlayer.
 					className={css.controlsHandleAbove}
-					onClick={this.showControls}
```

Comment:
Why was this removed? I found the offending commit (489bff1b978332c738d81c755ad5b160c608cef1) but not sure why it was included.

Reply:
It seems unnecessary to me. `SpottableDiv` is non-clickable as it's height is 0. `Overlay` handles `onVideoClick` to show controls.